output/plugins/additional/xml/additional.xml

```diff
@@ -81,16 +81,23 @@ Written by
     <property name="tab_width" type="text" help="The number of spaces per tab character.">4</property>
   </objectinfo>
 
-  <objectinfo class="wxToggleButton" startgroup="1" icon="toggle_button.xpm" type="widget">
+
+  <objectinfo class="wxToggleButton" icon="toggle_button.xpm" type="widget">
```

Comment:
Any specifiy reason why this element shouldn't start a group anymore? I think it makes even more sense now that there is an additional toggle-button